### PR TITLE
timeout strings

### PR DESCRIPTION
**Pull request by FnrDev**
**Please describe the changes this PR makes and why it should be merged:**

**In this PR:**

### CODE REVIEW

**Review by Dramex — CHANGES_REQUESTED** [review]
changes

**Comment by FnrDev** [comment]
All done

File: en/bot.json [hunk 1]
```diff
@@ -439,5 +439,13 @@
 	"slowmode_set": "This channel slowmode has been set to {0}.",
 	"slowmode_should": "Time should be less than or equal to {0}.",
 	"slowmode_is": "This channel slowmode is {0}.",
-	"slowmode_disabled": "Slowmode has been disabled from this channel."
+	"slowmode_disabled": "Slowmode has been disabled from this channel.",
+	"timeout_done": ":white_check_mark: **{0} timed out from the server!**",
+    "timeout_cant": ":rolling_eyes: - ** You can't timeout @{0}. **",
+    "timeout_missing_permission": ":rolling_eyes: - I couldn't timeout that user. Please check my permissions and role position.",
+	"timeout_max_time": "Timeout duration must be lower or equal to 28 days.",
```
Comment (Dramex) [inline en/bot.json:446]:
```suggestion
	"timeout_max_time": ":x: Timeout duration must be lower or equal to 28 days.",
```

File: en/bot.json [hunk 2]
```diff
@@ -439,5 +439,13 @@
 	"slowmode_set": "This channel slowmode has been set to {0}.",
 	"slowmode_should": "Time should be less than or equal to {0}.",
 	"slowmode_is": "This channel slowmode is {0}.",
-	"slowmode_disabled": "Slowmode has been disabled from this channel."
+	"slowmode_disabled": "Slowmode has been disabled from this channel.",
+	"timeout_done": ":white_check_mark: **{0} timed out from the server!**",
```
Comment (Dramex) [inline en/bot.json:443]:
```suggestion
	"timeout_done": ":white_check_mark: **{0} has been timed out!**",
```

File: en/bot.json [hunk 3]
```diff
@@ -439,5 +439,13 @@
...
-	"slowmode_disabled": "Slowmode has been disabled from this channel."
+	"slowmode_disabled": "Slowmode has been disabled from this channel.",
+	"timeout_done": ":white_check_mark: **{0} has been timed out!**",
+    "timeout_cant": ":rolling_eyes: - ** You can't timeout @{0}. **",
+    "timeout_missing_permission": ":rolling_eyes: - I couldn't timeout that user. Please check my permissions and role position.",
+	"timeout_max_time": ":x: Timeout duration must be lower or equal to 28 days.",
+	"untimeout_done": ":white_check_mark: **{0} has been untimed out.**",
+	"untimeout_cant": ":rolling_eyes: - ** You can't untimeout @{0}. **",
+	"untimeout_missing_permission": ":rolling_eyes: - I couldn't untimeout that user. Please check my permissions and role position.",
+	"untimeout_not_timedout": "❌ Member is not timed out."
```
Comment (Dramex) [inline en/bot.json:450]:
```suggestion
	"untimeout_not_timedout": ":x: Member is not timed out."
```

**Review by Dramex — APPROVED** [review]

**Pull request merged**